doc/article/en-US/dialog-basics.md

```diff
@@ -237,10 +241,28 @@ You can specify global settings as well for all dialogs to use when installing t
   </source-code>
 </code-listing>
 
-### Controller Settings
+### Dialog Settings
 
 The settings available for the dialog are set on the dialog controller on a per-dialog basis.
 - `viewModel` can be url, class reference or instance.
+  - url - path relative to the application root.
+    ```
+    .
+    +-- app.js
+    +-- forms
+        +-- consent-form.js
+        +-- consent-form.html
+    +-- prompts
+        +-- prompt.js
+        +-- prompt.html
+    ```
+    If you want to open a `prompt` from `consent-form` the path will be `prompts/prompt`.
+    > Warning: WebPack users should use `PLATFORM.moduleName` when passing a path. Not using it may lead to the resource not being bundled, when not referenced anywhere else.
```

Comment:
"when not referenced anywhere else"
It's more subtle than that. 
`moduleName` also ensures other properties important for `aurelia-loader`, like the assignment of a module id based on path, preserving exports and preventing their renaming, ensuring that the module doesn't get merged by `ModuleConcatenationPlugin`...

Bottom line: if a string is a dependency loaded by `aurelia-loader`, _always_ mark it as such with `moduleName`.